Core/MIPS/ARM64/Arm64CompVFPU.cpp

```diff
@@ -654,12 +654,58 @@ namespace MIPSComp {
 				switch ((op >> 23) & 7) {
 				case 2:  // vmin
 				{
+					fp.FCMP(fpr.V(sregs[i]), fpr.V(tregs[i]));
+					FixupBranch unordered = B(CC_VS);
 					fp.FMIN(fpr.V(tempregs[i]), fpr.V(sregs[i]), fpr.V(tregs[i]));
+					FixupBranch skip = B();
+
+					SetJumpTarget(unordered);
+					// Move to integer registers, it'll be easier.  Or maybe there's a simd way?
```

Comment:
Thought about that too.  We could move 7 instructions here (from TST to CSEL) at the cost of a CALL + const load + RET.  It would probably be better.

-[Unknown]